fetcher/floatingip.go

```diff
@@ -7,12 +7,14 @@ import (
 var _ Fetcher = &floatingIP{}
 
 // NewFloatingIP creates a new fetcher that will collect pricing information on floating IPs.
-func NewFloatingIP(pricing *PriceProvider) Fetcher {
-	return &floatingIP{newBase(pricing, "floatingip", "location", "type")}
+func NewFloatingIP(pricing *PriceProvider, additionalLabels ...string) Fetcher {
+	labels := append([]string{"location", "type"}, additionalLabels...)
+	return &floatingIP{newBase(pricing, "floatingip", labels...), additionalLabels}
 }
 
 type floatingIP struct {
 	*baseFetcher
+	additionalLabels []string
```

Comment:
I think this could be moved to the `baseFetcher` struct, to avoid repeating it on all fetcher types, right? This obviously applies to all other occurences as well, but I am just going to mark it here 😉

Reply:
Changed with https://github.com/jangraefen/hcloud-pricing-exporter/pull/48/commits/b443485798a87fa527b28798ca230b07afa58992